src/bundle/Controller/SearchController.php

```diff
@@ -212,4 +231,17 @@ public function searchAction(Request $request): Response
             'user_content_type_identifier' => $this->userContentTypeIdentifier,
         ]);
     }
+
+    /**
+     * @param string|null $languageCode
+     *
+     * @return array
+     */
+    private function getSearchLanguageFilter(?string $languageCode): array
+    {
+        return [
+            'languages' => $languageCode ? [$languageCode] : [],
```

Comment:
```suggestion
            'languages' => (array)$languageCode,
```

Reply:
If $languageCode will be empty string, the array with empty value will be created:

`$languageCode = '';

var_dump(
    (array)$languageCode
);`

Result:

`array(1) {
  [0]=>
  string(0) ""
}
`